index.js

```diff
@@ -46,7 +45,7 @@ module.exports = class ServeDrive extends ReadyResource {
     for (const c of this.connections) {
       c.destroy()
     }
-    if (this.server.listening) {
```

Comment:
What happens if the server was already closed for any reason? Should check for the `listening` property

Reply:
only closed here no?

Reply:
Does the `cb` in `server.close(cb)` resolves if already closed? If so then just checking `this.opened` as above should be ok

Reply:
why would it be closed?

Reply:
I forgot that defensive programming never ends